scripts/util/tcga_util.py

```diff
@@ -61,10 +63,24 @@ def get_args():
                         help='Remove mutation data from y matrix')
     parser.add_argument('-z', '--drop_rasopathy', action='store_true',
                         help='Decision to drop rasopathy genes from X matrix')
+    parser.add_argument( '--drop_x_genes', default=None,
```

Comment:
lets use [`nargs='+'`](https://docs.python.org/3/library/argparse.html#nargs).

This will load a list instead of a comma separated string. This will help [here](https://github.com/greenelab/pancancer/pull/98/files#diff-9a9e5b89778a2173d8d91e4cd0854c1aR189)

Reply:
also remove extra space